Lib/test/test_print.py

```diff
@@ -155,7 +155,7 @@ def test_string_with_excessive_whitespace(self):
 
         self.assertIn('print("Hello World", end=" ")', str(context.exception))
 
-    def test_string_with_stream_redirection(self):
+    def test_print_string_with_stream_redirection(self):
         import sys
         python2_print_str = 'print >> sys.stderr, "message"'
```

Comment:
@ncoghlan  But that is just a string, we would need to execute it in the context manager, no?